Spanner/tests/Unit/SpannerClientGcpOptimizerEnabledTest.php

```diff
@@ -0,0 +1,310 @@
+<?php
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Spanner\Tests\Unit;
+
+require_once (__DIR__ . "/../../../vendor/autoload.php");
```

Comment:
This shouldn't be required if you're running tests from the project root with the provided phpunit configuration file.

Reply:
Done.